docs/MSPX.md

```diff
@@ -19,33 +22,92 @@ mLRS provides the MspX technology, which is designed to improve the over-the-air
+
+## Connection Types with different INAV Versions
+
+Depending on the chosen connection and serial mode, the functionality will slightly differ, especially in INAV versions before 8. This is a quick overview, what options are available and what functions to expect. 
+
+### INAV 8
+
+Serial Mode:
+- MspX
+  - provides telemetry sensors to your radio
+  - allows RC Control over MSP (rc override)
+  - Full MSP GCS Functionality with INAV Configurator, MWP and other applications
+  - comprehensive OSD and Blackbox Link Statistics with dedicated MSP messages provided by mLRS
+  - limited to 50Hz RC Rate (Only relevant for 2.4GHz FLRC Mode with 111Hz Packet rate, Will be limited to 37Hz RC rate)
+- Transparent
+  - See INAV 7 section
+  - Not recommended
+
+RC Out Mode (Can be combined with Serial if the Rx has 2 UART)
+- CRSF
+  - allows RC Control
+  - RC Rate up to 111Hz in 2.4GHz FLRC mode
+
+ ***Recommendation***
+ 
+Since INAV 8 in combination with mLRS can provide all features over MspX, it is recommended to only use a single UART MspX connection for most vehicles. The only exception is the 2.4GHz FLRC mode that can provide 111Hz packet and RC rate. 
+Due to a performance limitation in INAV, this mode had to be limited to 37Hz RC Rate over MSP. Use a separated CRSF UART instead, to achieve full 111Hz RC Update Rates in this mode. All other features will work as normal. 
+
+### INAV 7
+
+Serial Mode: 
+- MspX
+  - provides telemetry sensors to your radio
+  - allows RC Control over MSP (rc override)
+  - limited to 50Hz RC Rate (Only relevant for 2.4GHz FLRC Mode with 111Hz Packet rate, Will be limited to 37Hz RC rate)
+  - Ground control station use is limited to applications that can handle unrequested MSP messages (MWPTools only works in Monitor-Mode, INAV Configurator will be slow to load parameters, etc.)
+  - Will not provide RC Link Statistics for OSD or Blackbox
+- Transparent
+  - Full functionality of GCS Software but possibility of higher message loss (Especially big MSP Messages like ADS-B Data or Waypoint Mission transfers)
+  - No RC Control, No Telemetry to the radio (needs CRSF for RC control) 
+  - No link statistics for the OSD (Needs CRSF for OSD Link Information)
+
+RC Out Mode (Can be combined with Serial if the Rx has 2 UART)
+- CRSF
+  - allows RC Control
+  - provides link statistics for OSD and Blackbox
+  - No Telemetry
+
+ ***Recommendation*** 
+ 
+ For INAV 7 we recommend one of three options, depending on the usecase. 
+ - MspX+CRSF if mLRS is only used as a radio control system for FPV. The separate CRSF for RC Control is recommended, to have OSD Link Statistics available
+ - MspX+MSP-RC if no OSD link statistics are needed (autonomous flights and LOS only) and if you use a GCS Software that can handle passive MSP Message Monitoring with no 2-way communication (MWP)
```

Comment:
Should this be 'or' instead of 'and' ?

Reply:
'and' was intentional as otherwise not even MspX makes no sense.